src/EventSubscriber/HttpCacheResponseSubscriber.php

```diff
@@ -39,9 +39,13 @@ public static function getSubscribedEvents()
 
     public function configureCache(FilterResponseEvent $event)
     {
-        $view = $event->getRequest()->attributes->get('view');
-        if (!$view instanceof CachableView || !$view->isCacheEnabled()) {
-            return;
+        if ($event->getRequest()->attributes->has('is_rest_request') && $event->getRequest()->attributes->get('is_rest_request') === true) {
```

Comment:
Maybe needs a new name, but this subscriber to me is for ContentView.

Not all rest requests are supposed to be cacheable afaik, so if (we make it) expose RestValue or something on Response _(is it maybe still on the getControllerResult?)_, then we can have a dedicated subscriber for rest responses.

Reply:
see #25 I would think this part is not needed anymore if we handle `X-Location-Id` issue in separate listener

Reply:
+1